UI Actions/Close child incident/clientScript.js

```diff
@@ -0,0 +1,6 @@
+function popUpClientScript(){
+	var answer = confirm('Are you sure you want to close the attached child incident'); // confirm message to ask for confirmation
```

Comment:
Please note that using confirm and similar basic Javascript Window methods (such as alert, prompt, etc.) is not recommended in ServiceNow (amongst other limitations, these windows are rendered by user's browser and not by ServiceNow). I would encourage to explore the [GlideModal API ](https://developer.servicenow.com/dev.do#!/reference/api/vancouver/client/c_GlideModalV3API)as a replacement for future use (there are many out of the box UI page to use there, including at least 2 for confirm).
I will accept to code as it is technically correct, but if you'd like, you are welcome to commit an update to it using GlideModal!